src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AzureSqlFailoverGroupCmdletBase.cs

```diff
@@ -25,16 +25,6 @@ namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
     public abstract class AzureSqlFailoverGroupCmdletBase : AzureSqlCmdletBase<IEnumerable<AzureSqlFailoverGroupModel>, AzureSqlFailoverGroupAdapter>
     {
         /// <summary>
-        /// Gets or sets the name of the server to use.
-        /// </summary>
-        [Parameter(Mandatory = true,
```

Comment:
@dlemMSFT any reason for removing this parameter from the base cmdlet if you are just going to add it back to each cmdlet that inherits from it?